docs/fr/style.mdx

```diff
@@ -93,6 +140,10 @@ function generateModule(moduleType: MODULE_TYPE) {
 }
 ```
 
+:::caution Attention
+On **DEVRAIT toujours assigner une valeur** à la clé de l'enum afin d'éviter des effets indésirables.
```

Comment:
Tu peux linker ce blog post pour comprendre pourquoi : https://blog.beraliv.dev/2022-09-10-with-or-without-enums#numeric-enums-are-not-type-safe


```suggestion
On **DEVRAIT toujours assigner une valeur** à la clé de l'enum afin d'éviter des effets indésirables (voir pourquoi dans [cet article (En)](https://blog.beraliv.dev/2022-09-10-with-or-without-enums#numeric-enums-are-not-type-safe)).
```